README.md

```diff
@@ -1,4 +1,5 @@
 ﻿<!-- markdownlint-disable-next-line -->
+
```

Comment:
Please remove this line, otherwise, the Markdown Linter Ignorer won't work.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient